docs/src/get-started/monitor-and-troubleshoot/log.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Git log
+weight: -10
+description: See what's happening inside your app.
+---
+
+Once your app is up and running, you want to monitor it to make sure it stays that way.
+Take advantage of the observability of apps running on Platform.sh to see everything that's happening.
+
+## Check activities
+
+All events that change your environments are logged as activities.
+See a list of all activities by running the following command:
+
+```bash
+platform activities
+```
+
+You get a table with all activities that have run or are running.
+The most recent one at the top of the table is probably your merge of the `dev` branch into `main`.
+
+To see all the details of that activity, copy its `ID` from the table.
+Then use the ID in place of `<ID>` in the following command:
+
+```bash
+platform activity:get <ID>
+```
+
+This returns the build log from the merge activity.
+This way, you can keep track of everything that happens in your project.
+
+## View logs
+
+Another way to keep track of your project and troubleshoot any issues is to view its logs.
+There are different types of logs you can view, such as error logs and any logs your app creates.
```

Comment:
It's a general suggestion, especially to avoid multiple repetitions, rather than a specific rule that always needs to be followed. The original wasn't so bad, but I can try to rephrase.